systemFlake.nix

```diff
@@ -145,29 +124,35 @@ let
   );
 
 in
-otherArguments
-
-// eachSystem supportedSystems (system:
-  let
-    importChannel = name: value: import (patchChannel system value.input (value.patches or [ ])) {
-      inherit system;
-      overlays = sharedOverlays ++ (if (value ? overlaysBuilder) then (value.overlaysBuilder pkgs) else [ ]);
-      config = channelsConfig // (value.config or { });
-    };
-
-    pkgs = mapAttrs importChannel channels;
-
-    optional = check: optionalAttrs (check != null);
-  in
-  { inherit pkgs; }
-  // optional packagesBuilder { packages = packagesBuilder pkgs; }
-  // optional defaultPackageBuilder { defaultPackage = defaultPackageBuilder pkgs; }
-  // optional appsBuilder { apps = appsBuilder pkgs; }
-  // optional defaultAppBuilder { defaultApp = defaultAppBuilder pkgs; }
-  // optional devShellBuilder { devShell = devShellBuilder pkgs; }
-  // optional checksBuilder { checks = checksBuilder pkgs; }
-)
+mergeAny otherArguments (
```

Comment:
This isn't necessary, but it allows for the `nixosConfigurations` escape hatch. You can set hosts via that and they get merged with the ones created by fup. Also could be useful if there is ever a top-level overlaysBuilder.

Either way its perfectly safe and I thought it would just be good to go ahead and merge otherArguments with fups code properly.